provisioning/salt-cloud.md

```diff
@@ -1,53 +0,0 @@
-# Provisioning using Salt Cloud
```

Comment:
could you please also update the SUMMARY.md when we remove entire MD file in order to not generate HTML from GitBook please?